src/utils/coinSelection.spec.js

```diff
@@ -331,7 +331,7 @@ describe('Utils - coinSelection', function suite() {
 
       while(inputValue<outputValue){
         if(copiedUtxos.length === 0){
-          throw new Error('Not enought utxo');
+          throw new Error('Not enought UTXOs');
```

Comment:
Oops, missed this the first time :see_no_evil: 
```suggestion
          throw new Error('Not enough UTXOs');
```

Reply:
It's already merged, but I do will update that on another PR, I've taken note of it on my todo :) Thank you :)